### PR TITLE
feat!: Return Pydantic models instead of dicts in UtaDatabase

### PR DESCRIPTION
**Pull request by korikuzma**
* Only focused on `UtaDatabase` for now
* `alt_pos_change` was renamed to `pos_change`

Created `staging` branch for these changes, just in case other issues wanted to be addressed / released.

### CODE REVIEW

File: src/cool_seq_tool/schemas.py
[the remark this answers is not in the document]
Comment (jsstevenson) [reply src/cool_seq_tool/schemas.py:1]: Could also consider taking the pydantic models that are associated with specific modules (eg db connection args w/ the uta handler) and putting them directly into that module. The pattern of a distinct schemas.py module or group of modules is more suited to MVC-like web apps imho (where you have a models module, a views module, and a controller module), and while there are some models that are truly general-purpose and can't be placed anywhere else, others can probably be moved closer to where they're used.

[the remark this answers is not in the document]
Comment (korikuzma) [reply src/cool_seq_tool/schemas.py:1]: @jsstevenson I had originally done that here and in a different PR, but I moved it to schemas to be consistent with current approach. I've been going back and forth. I don't have a strong preference. Do you?

[the remark this answers is not in the document]
Comment (jsstevenson) [reply src/cool_seq_tool/schemas.py:1]: Yeah, I think it's generally good to be adhering to collocation/"the principle of proximity", at least in cases where it's easy and non-disruptive. Web API response models often live in their own modules by convention but I don't think that needs to be true for all function arguments and responses.

[the remark this answers is not in the document]
Comment (korikuzma) [reply src/cool_seq_tool/schemas.py:1]: Sounds good! 👍 Probably good to check we're doing this in other projects as well

[the remark this answers is not in the document]
Comment (korikuzma) [reply src/cool_seq_tool/schemas.py:1]: Also I forgot you're PTO! No need to respond to GitHub comments/PRs/etc.

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**